src/graphql.js

```diff
@@ -845,5 +845,23 @@ async function anilistAPI(query, queryArgs){
 	console.error(`AniList API returned ${res.status} ${res.statusText}`)
 	return null;
 }
+
+/**
+ * Runs an API cache checkup weekly
+ * @returns {Promise|null}
+ */
+const cacheCheckup = async () => {
+	const check = localStorage.getItem("automail-db-check")
+	if(check){
+		if(NOW() > check){
+			localStorage.setItem("automail-db-check", NOW()+7*24*60*60*1000)
+			return await flushCache();
```

Comment:
Why is this needed?

Reply:
Requests that have been cached with an expiry date wouldn't get removed until the same call is made again. I was imagining a scenario where some of these expired cache items would continue to accumulate without ever being "flushed", so the solution to that would either be an automated "checkup" that periodically iterates through and removes the expired items or a button in the automail settings that a user could manually "flush" the database.

Was inspired by this bit of code here:
https://github.com/hohMiyazawa/Automail/blob/9ac16a628e3bed9389b32d09e910e3111a863bd3/src/controller.js#L403-L412

Reply:
Okay, makes sense. Expiration time can be tweaked later as needed.